raiden/api/rest.py

```diff
@@ -1028,12 +1033,20 @@ def initiate_payment(
                 target=target_address,
                 amount=amount,
                 identifier=identifier,
+                secret=secret,
+                secret_hash=secret_hash,
             )
-        except (InvalidAmount, InvalidAddress, PaymentConflict, UnknownTokenAddress) as e:
+        except (InvalidAmount, InvalidAddress,
```

Comment:
style: Once you start splittng a list of arguments in multiple lines we want them each in its own line:
source: [here](https://github.com/raiden-network/raiden/blob/master/CONTRIBUTING.md)

Reply:
Done